src/packages/sdk/src/predefinedGeneratorResolvers.ts

```diff
@@ -44,6 +45,24 @@ async function getPrismaClientDir(
   )[0]
 }
 
+async function getPrismaClientDir(baseDir: string) {
+  // we check that the found client is not the one of the CLI
+  const clientDir = await getPkgDir(baseDir, '@prisma/client')
+  const cliDir = await getPkgDir(baseDir, 'prisma')
+
+  // if we are testing the client in the CLI folder
+  if (cliDir && process.cwd().includes(cliDir)) {
```

Comment:
Should this not be `&& process.cwd().includes(clientDir)` or am i missing something?

Reply:
Didn't read the comment :see_no_evil: